src/Athena/Views/Home/NewAccountPage.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿
+@{
+    ViewData["Title"] = "NewAccountPage";
+}
+
+<h2>Welcome to the Athena Scheduler!</h2>
+
+<div class="container-fluid">
+
+    <!-- Dropdown Trigger -->
+    <a class='dropdown-button btn' href='#' data-activates='dropdown1'>Select Your University</a>
+
+    <!-- Dropdown Structure -->
+    <ul id='dropdown1' class='dropdown-content'>
+        <li><a href="#!">University one</a></li>
+        <li><a href="#!">University two</a></li>
+    </ul>
+
+    <a href="Main_Page" style="color:white" class="btn indigo darken-4">Create Account</a>
+    <a href="Home" style="color:white" class="btn">Cancel</a>
```

Comment:
Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.